ceph.spec.in

```diff
@@ -14,7 +14,7 @@
 #
 # Please submit bugfixes or comments via http://tracker.ceph.com/
 # 
-%bcond_with ocf
+%bcond_without ocf
```

Comment:
@tchaikov Actually, it already is called `ceph-resource-agents` due to the way RPM parses spec files. To make it be `resource-agents` there would need to be a `-n` in this line:

```
%package resource-agents
```

But, yes, in openSUSE the ocf build is enabled.